components/ModPagination.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+    <div class="flex items-center justify-center gap-2 mt-4">
+        <!-- First page -->
+        <button
+            :disabled="currentPage === 1"
+            class="w-10 h-10 flex items-center justify-center rounded bg-gray-400 disabled:opacity-30 disabled:cursor-not-allowed"
```

Comment:
instead of `bg-gray` and `text-white` let's use our theme colors. 
Otherwise, the gray won't look right when we use a dark theme or high-contract them for example